Source/Core/Core/Slippi/SlippiNetplay.h

```diff
@@ -44,6 +44,8 @@ class SlippiPlayerSelections
 	u8 playerIdx = 0;
 	u8 characterId = 0;
 	u8 characterColor = 0;
+	u8 teamId = 0;
```

Comment:
This needs to get added to the logic in `Merge()` below

Reply:
Done